app/models/data_update_script.rb

```diff
@@ -3,15 +3,14 @@ class DataUpdateScript < ApplicationRecord
   NAMESPACE = "DataUpdateScripts".freeze
   STATUSES = { enqueued: 0, working: 1, succeeded: 2, failed: 3 }.freeze
 
-  default_scope { order(file_name: :asc) }
-
   enum status: STATUSES
   validates :file_name, uniqueness: true
 
   class << self
     def scripts_to_run
-      ids = load_script_ids
-      enqueued.where(id: ids)
+      insert_new_scripts
+
+      enqueued.where(id: ids).order(file_name: :asc)
```

Comment:
@rhymes The assignment to `ids` got removed but it's still used in this `where` clause. Maybe this?

```ruby
ids = insert_new_scripts
enqueued.where(id: ids).order(file_name: :asc)
```